rabbitmq-ballerina/build.gradle

```diff
@@ -293,7 +320,7 @@ unpackStdLibs.dependsOn unpackJballerinaTools
 copyStdlibs.dependsOn unpackStdLibs
 updateTomlVersions.dependsOn copyStdlibs
 ballerinaBuild.dependsOn updateTomlVersions
-ballerinaTest.dependsOn startRabbitmqServer
-ballerinaBuild.dependsOn ":rabbitmq-native:build"
-ballerinaTest.finalizedBy stopRabbitmqServer
+ballerinaBuild.dependsOn test
+ballerinaBuild.dependsOn startRabbitmqServer
+ballerinaBuild.finalizedBy stopRabbitmqServer
```

Comment:
```suggestion
ballerinaBuild.dependsOn startRabbitmqServer
ballerinaBuild.finalizedBy stopRabbitmqServer
ballerinaTest.dependsOn startRabbitmqServer
ballerinaTest.finalizedBy stopRabbitmqServer
```

Reply:
Do we need this fix?

Reply:
Yes, if we run `./gradlew test` the `ballerinaBuild` task will not run, the `ballerinaTest` task will run instead.